### ISSUE TITLE
Rojo returns an error when serving from Ctrl+Shift+P > Rojo: Open menu

### ISSUE THREAD

**Issue by Infraction7** [rojo-rbx/rojo#814]
Error log from VSC output:
`[ERROR rojo] Details: error binding to 127.0.0.1:34872: error creating server listener: Обычно разрешается только одно использование адреса сокета (протокол/сетевой адрес/порт). (os error 10048)
[ERROR rojo] in file C:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\hyper-0.14.27\src\server\server.rs on line 79`

**Comment by Dekkonot** [comment]
That error tends to happen when you have multiple versions of Rojo running. There's a few other extensions that might run Rojo, most notably Luau LSP; can you confirm that you don't have any other copies running in the background using task manager?

**Comment by Infraction7** [comment]
I dont have other rojo's running but i have Roblox LSP, API Explorer and Roblox Lua Autocompletes. May one of them cause the issue?

**Comment by Dekkonot** [comment]
I don't believe so, but it's not impossible. Does this persist through a system restart?
